routing/routing_session.cpp

```diff
@@ -156,17 +158,17 @@ void RoutingSession::RebuildRouteOnTrafficUpdate()
 
     switch (m_state)
     {
-    case RoutingNotActive:
-    case RouteNotReady:
-    case RouteFinished: return;
+    case SessionState::RoutingNotActive:
+    case SessionState::RouteNotReady:
+    case SessionState::RouteFinished: return;
 
-    case RouteBuilding:
-    case RouteNotStarted:
-    case RouteNoFollowing:
-    case RouteRebuilding: startPoint = m_checkpoints.GetPointFrom();
+    case SessionState::RouteBuilding:
+    case SessionState::RouteNotStarted:
+    case SessionState::RouteNoFollowing:
+    case SessionState::RouteRebuilding: startPoint = m_checkpoints.GetPointFrom();
```

Comment:
please add break

Reply:
Are you talking about something link that
```c++
    switch (m_state)
    {
    case SessionState::RoutingNotActive:
    case SessionState::RouteNotReady:
    case SessionState::RouteFinished: return;

    case SessionState::RouteBuilding:
    case SessionState::RouteNotStarted:
    case SessionState::RouteNoFollowing:
    case SessionState::RouteRebuilding:
      startPoint = m_checkpoints.GetPointFrom();
      break;

    case SessionState::OnRoute:
    case SessionState::RouteNeedRebuild: break;
    }
```

Reply:
?

Reply:
Done

Reply:
correct